src/main/scala/higherkindness/rules_scala/common/sbt-testing/SubprocessRunner.scala

```diff
@@ -10,14 +10,14 @@ object SubprocessTestRunner {
   def main(args: Array[String]): Unit = {
     val input = new ObjectInputStream(System.in)
     val request = input.readObject().asInstanceOf[TestRequest]
-
+    val testFrameworkArguments = Seq.empty[String]
```

Comment:
Can you add a `testArgs` field to `TestRequest` so that they are available here? You can pass them in when the `TestRequest` is created in `ProcessTestRunner.execute`